zetaclient/testutils/os.go

```diff
@@ -0,0 +1,15 @@
+package testutils
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// create a temporary directory for testing
+func CreateTempDir(t *testing.T) string {
+	tempPath, err := os.MkdirTemp("", "tempdir-")
+	require.NoError(t, err)
+	return tempPath
+}
```

Comment:
NIT: Why not put this is the sample directory?

Reply:
https://github.com/zeta-chain/node/pull/2357/commits/e585bc0249a23d140a11e94c4be327d0075a6e8c